### ISSUE TITLE
Shared Crosshair concept

### ISSUE THREAD

**Issue by Huey-H** [swimlane/ngx-charts#1184]
I am using the line graphs and realize it'd be nice if there is a feature where if there are multiple of the same graph being used within the same context (running through for loop to draw multiple graphs), when hovering over one of the graph's x-axis, the other graphs will be hovered over showing the value of the respective data in that graph

**Comment by app789** [comment]
Yes, that would be a very useful feature. ng2-charts is a having a similar vertical and horizontal crosshairs which we can move with mouse.
